hw/ip/kmac/data/kmac.hjson

```diff
@@ -0,0 +1,340 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+{ name: "kmac"
+  clock_primary: "clk_i"
+  bus_device: "tlul"
+  bus_host: "none"
+
+  interrupt_list: [
+    { name: "kmac_done"
+      desc: "KMAC/SHA3 absorbing has been completed"
+    }
+    { name: "fifo_empty"
+      desc: "Message FIFO empty condition"
+    }
+    { name: "kmac_err"
+      desc: "KMAC/SHA3 error occurred. ERR_CODE register shows the details"
+    }
+  ]
+  alert_list: []
+  param_list: [
+    { name:    "NumWordsKey"
+      type:    "int"
+      default: "16"
+      desc:    "Number of words for the secret key"
+      local:   "true"
+    }
+    { name: "NumWordsPrefix"
+      type: "int"
+      default: "11"
```

Comment:
do you think we need to run this by software? They probably won't have much of an opinion now, but might be worth just dropping an issue so we don't forget to get their input later.

Reply:
OK. I will share preliminary hw ip spec to sw ppl to look into it.